JMeterAKSPipeline/CreateLoadTestInfra.ps1

```diff
@@ -0,0 +1,47 @@
+param(
+    [Parameter(Mandatory = $True)]
+    [string]$tenant,
+    [Parameter(Mandatory = $True)]
+    [string]$defaultNamespace,
+    [Parameter(Mandatory = $True)]
+    [string]$resourceGroup,
+    [Parameter(Mandatory = $True)]
+    [string]$aksClusterName,
+    [Parameter(Mandatory = $True)]
+    [string]$spnClientId,
+    [Parameter(Mandatory = $True)]
+    [string]$spnClientSecret,
+    [Parameter(Mandatory = $True)]
+    [string]$aksRegion,
+    [Parameter(Mandatory = $False)]
+    [string]$nodeVmSize = "Standard_D2s_v3"
+)
+
+function log([string] $message, [string] $color) {
+    Write-Host "$(get-date) $message"   -ForegroundColor $color 
+    Write-Host " " 
+}
+
+#################### Creating AKS Cluster ################################################
+
+try {
+    log "############# Installing Extension : aks-preview ###############" "DarkYellow"
```

Comment:
we are loosing readability with function here, i think we should directly use write-host with foreground color option here. with function we are limited with write-host, pwsh gives you other commands which we can use for printing i.e. write-information write-error or so.

Reply:
This function is printing with time stamp and is useful to avoid this repetition.
For exception handling we have Write-Error powershell commands.

Since most of the logging is Write-Host this function comes in handy.